Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -67,7 +67,7 @@ - (NSArray *)managedObjectContext:(NSManagedObjectContext *)context
             finishWithStatus:result == nil ? kSentrySpanStatusInternalError : kSentrySpanStatusOk];
 
         SENTRY_LOG_DEBUG(@"SentryCoreDataTracker automatically finished span with status: %@",
-            error == nil ? @"ok" : @"error");
+            result == nil ? @"error" : @"ok");
```

Comment:
I swapped the order of the parameters here to make the logic in the logging mirror the logic in the `[span finishWithStatus:]` call above.

Previously, this logging was almost always incorrect; it was just checking whether the user had passed in an `error` pointer. If so, it logged "error".